verilog/formatting/align.cc

```diff
@@ -1044,6 +1044,40 @@ std::function<MemberType(const StructType&)> function_from_pointer_to_member(
 using AlignmentHandlerMapType =
     std::map<AlignableSyntaxSubtype, AlignmentGroupHandlers>;
 
+std::vector<verible::ColumnPositionEntry> non_tree_column_scanner(
+    verible::TokenRange token_range, const verible::SyntaxTreeNode& node,
+    long unsigned int max_children_size) {
+  std::vector<verible::ColumnPositionEntry> trailing_column_entries;
+  auto tag = NodeEnum(node.Tag().tag);
+  VLOG(2) << __FUNCTION__ << ", node: " << tag;
+
+  for (auto token : token_range) {
+    switch (token.token_enum()) {
+      case 44: {
```

Comment:
44? can we use proper `TK_` define?

Reply:
Actually I don't see any `TK_` macro for `,`. I added a comment that 44 is `,`

Reply:
If no define/enum is generated for this then we need to make sure to add that generation, using a literal here seems pretty fragile

Reply:
I just realized 44 is ascii for comma.

It should be okay to use
```
case ',':
```
then

Reply:
Fixed